pkg/build/pipelines/perl/cpan-install.yaml

```diff
@@ -0,0 +1,11 @@
+name: Install a Perl package downloaded from cpan
```

Comment:
Nit, but the way I read the pipeline, I understood that it just creates the Makefile that you then will have to use in `make` later

Reply:
That's true! I'll rename.